src/components/Badges/Badges.tsx

```diff
@@ -0,0 +1,42 @@
+import { colors } from '../../typography/colors';
+import { Caption } from '../typography/caption/caption';
+import './index.css';
+import { ReactNode, useState } from 'react';
+
+export interface PropsBadges {
+  title: string;
+  icon: ReactNode;
+  onClick: () => void;
+  selected: boolean;
+}
+
+export function Badges({ title, icon, onClick, selected }: PropsBadges) {
```

Comment:
```suggestion
export function Badge({ title, icon, onClick, selected }: PropsBadges) {
```

O componente representa uma badge, não várias, certo?